interface/portraits/portraits_kvaldir_sprites.gfx

```diff
@@ -40,530 +26,33 @@ spriteTypes = {
-		noOfFrames = 15
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_hair_behind_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_hair_behind_3.dds"
-		noOfFrames = 15
-		norefcount = yes
-	}
-	
-	spriteType = {
-		name = "GFX_kvaldir_male_cheeks"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_cheeks_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-	
-	spriteType = {
-		name = "GFX_kvaldir_male_cheeks_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_cheeks_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-	
-	spriteType = {
-		name = "GFX_kvaldir_male_cheeks_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_cheeks_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_neck"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_neck_1.dds"
-		noOfFrames = 4
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_neck_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_neck_2.dds"
-		noOfFrames = 4
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_neck_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_neck_3.dds"
-		noOfFrames = 4
-		norefcount = yes
-	}
-
-
-
-	spriteType = {
-		name = "GFX_kvaldir_male_chin"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_chin_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_chin_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_chin_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_chin_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_chin_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_mouth"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_mouth_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_mouth_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_mouth_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_mouth_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_mouth_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_nose"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_nose_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-	
-	spriteType = {
-		name = "GFX_kvaldir_male_nose_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-
 	spriteType = {
 		name = "GFX_kvaldir_male_eyes"
```

Comment:
Kvaldir female beards and hairs are here, why so?

Reply:
what?

Reply:
Open this file. Kvaldir don't have separate hair and beard layers.

Reply:
they dont have hair and beards

Reply:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77100881-c8050200-6a2f-11ea-90e8-1b43cabd99a9.png)

</details>